rmw_cyclonedds_cpp/src/rmw_node.cpp

```diff
@@ -1078,6 +1166,8 @@ static bool check_create_domain(dds_domainid_t did, rmw_localhost_only_t localho
       return false;
     }
 
+    RCUTILS_LOG_INFO_NAMED("rmw_cyclonedds_cpp", "Config XML is %s", config.c_str());
```

Comment:
This doesn't seem like a log info we should leave in, right?

Reply:
Changed to DEBUG in 8324e11